.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
     rev: v0.931
     hooks:
       - id: mypy
-        additional_dependencies: [torch==1.9.0, tokenize-rt==3.2.0, types-PyYAML, types-mock]
+        additional_dependencies: [torch, tokenize-rt==3.2.0, types-PyYAML, types-mock]
```

Comment:
Had to add this because `so3` was giving me a mypy error.  `torch.pi` was only added in torch 1.10, but pre-commit hooks was using `1.9.0`. @fantaosha you never saw this error?

Reply:
Strange though since all the CI checks passed in 286.

Reply:
I think this only affects pre-commit hooks, not CI (which uses `nox`).

Reply:
Ah okay. If this is fixed here then we are set. I suspect some of these will need updating based on the vmap PRs.